modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-web/test/js/util/FormURL.es.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import FormURL from '../../../src/main/resources/META-INF/resources/admin/js/util/FormURL.es';
+
+const formInstanceId = 1;
+
+describe('FormURL', () => {
+	it('gets a blank url', () => {
+		const formURL = new FormURL(1, false);
```

Comment:
@marcosapmf, why didn't you use the `formInstanceId` as the first parameter of the constructor? Wouldn't it be more clear to the reader instead of a number, like `1`?

Additionally, why did you just use two parameters instead of three? The parameters 2 and 3 have default values, thus why you just used one of them here? Can you please clarify?

Reply:
Hey @natocesarrego! 
Sorry about the magic number, I created `formInstanceId ` to avoid this on `expected` and I was blind about the FormURL constructor calls.
About the third parameter, FormURL has a default value on its constructor that's the cause I didn't put it here. But I think it's not so readable, right?

Reply:
Yeah. 😁